.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
+    "extends": ["airbnb","prettier"],
+    "plugins": ["prettier"],
+    "ignorePatterns": ["/tests/*.test.js"],
+    "rules": {
+      "prettier/prettier": ["error"]
+    }
+}
```

Comment:
Add new line to the end of all your files

Reply:
That is already in the default prettier settings